src/contexts/UserContext.js

```diff
@@ -10,126 +10,131 @@ const UserContext = React.createContext({
   state: null,
   district: null,
   representatives: null,
+  news: null,
   setError: () => {},
   clearError: () => {},
   setUser: () => {},
   setRepresentatives: () => {},
   processLogin: () => {},
-  processLogout: () => {},
-})
+  processLogout: () => {}
+});
 
-export default UserContext
+export default UserContext;
 
 export class UserProvider extends Component {
   constructor(props) {
-    super(props)
+    super(props);
     const state = {
```

Comment:
do we need news here too?

Reply:
> Code works. Should we set the news on a per-representative page or in the dashboard? Maybe up for a bigger discussion

Our dashboard I think needs more content, and this API call is returning articles about all 3 of our representatives, so makes sense on the dashboard